tested/dsl/schema.yaml

```diff
@@ -17,13 +17,13 @@ definitions:
       config:
         $ref: "#/definitions/globalConfig"
         description: "Configuration settings at plan level"
-      disable_optimizations:
+      disableOptimizations:
         type: "boolean"
         description: "Disable performance optimizations"
       namespace:
         type: "string"
         default: "submission"
-        description: "Namespace of the submitted solution, best in `snake_case`"
+        description: "Namespace of the submitted solution, best in `snakeCase`"
```

Comment:
Here I would put `snake case` in two words, as it's written in English, as the writing `snakeCase` is extra confusing.

Reply:
This is fixed in commit `f6a8203` as `snake_case` for the clearity.